modules/ROOT/pages/xamarin/using-metrics-sdk.adoc

```diff
@@ -0,0 +1,24 @@
+= AeroGear Services Metrics SDK
+
+This document explains how to use the AeroGear Metrics SDK on the Android platform and integrate it with an App.
+
+== Provided metrics
+
+The default metrics contain App metrics, Device metrics and possibly other types in the future.
```

Comment:
best to remove this "and possibly other types in the future" so it reads something like:
 - "The default metrics contain mobile app and mobile device metrics"